cpp/src/io/parquet/writer_impl.cu

```diff
@@ -1245,13 +1195,30 @@ void writer::impl::encode_pages(hostdevice_2dvector<gpu::EncColumnChunk>& chunks
   switch (compression_) {
     case parquet::Compression::SNAPPY:
       if (nvcomp_integration::is_stable_enabled()) {
-        snappy_compress(comp_in, comp_out, comp_stats, max_page_uncomp_data_size, stream);
+        nvcomp::batched_compress(nvcomp::compression_type::SNAPPY,
+                                 comp_in,
+                                 comp_out,
+                                 comp_stats,
+                                 max_page_uncomp_data_size,
+                                 stream);
       } else {
         gpu_snap(comp_in, comp_out, comp_stats, stream);
       }
       break;
-    default: break;
+    case parquet::Compression::ZSTD:
+      if (nvcomp_integration::is_all_enabled()) {
+        nvcomp::batched_compress(nvcomp::compression_type::ZSTD,
+                                 comp_in,
+                                 comp_out,
+                                 comp_stats,
+                                 max_page_uncomp_data_size,
+                                 stream);
+      }
+      break;
+    default: CUDF_FAIL("invalid compression type");
```

Comment:
This change to the default is causing failures in the java unit tests if the compression type is NONE.
```
[ERROR] testParquetWriteToFileUncompressedNoStats  Time elapsed: 0.018 s  <<< ERROR!
ai.rapids.cudf.CudfException: cuDF failure at: /home/jimb/git/spark-rapids-jni/thirdparty/cudf/cpp/src/io/parquet/writer_impl.cu:1218: invalid compression type
	at ai.rapids.cudf.Table.writeParquetChunk(Native Method)
	at ai.rapids.cudf.Table.access$300(Table.java:41)
	at ai.rapids.cudf.Table$ParquetTableWriter.write(Table.java:1278)
	at ai.rapids.cudf.TableTest.testParquetWriteToFileUncompressedNoStats(TableTest.java:7476)

```